feature/sessions/src/main/java/io/github/droidkaigi/confsched2023/sessions/SearchScreenViewModel.kt

```diff
@@ -78,4 +118,10 @@ class SearchScreenViewModel @Inject constructor(
             },
         )
     }
+
+    fun updateBookmark(timetableItem: TimetableItem) {
```

Comment:
It's my preference but how about using `onClickTimetableItemBookmark` to promote the best practice that notifying events to ViewModel.